lips/lip-10.md

```diff
@@ -73,6 +75,32 @@ struct LibraIdDomain {
 * The Libra ID domain can be created as part of creating a `ParentVASP` account.
 * An entity (potentially the Association) may at some point choose to expose an open source application(*Indexer*) that will allow indexing available Libra IDs and could provide a convenient REST API for applications to fetch information about the domains. The API of such an indexer is out of scope of this RFC.
 
+## Libra ID domain events
+
+Blockchain contract that manages Libra ID domains must emit an event every time Libra ID domain is created, removed or updated. Those events are critical for applications to be able to efficiently index existing Libra ID domains.
```

Comment:
Rewrite this and the following paragraph:

The blockchain contract that manages Libra ID domains emits an event every time a Libra ID domain is created, removed, or updated. These events are published under the Treasury Compliance account, whereas the actual Libra ID domains are published under VASP accounts. Applications can efficiently index Libra ID domains by querying these events conveniently from the Treasury Compliance account.